packages/salesforcedx-webview-ui/README.md

```diff
@@ -0,0 +1,122 @@
+# Introduction
```

Comment:
This is the main file that you should read to get the gist of this PR.
The other parts I will illustrate to the team via an internal demo.

Reply:
@lcampos - Would appreciate your thoughts on this. 

We can still merge this in and we can continue the discussion later but it would be appreciated if you could share some thoughts on this based on your work with sutro - I'm not sure if you were using webviews there (or browserview).

Either way, the links in the README.md should be useful.

Reply:
I did look at using webviews but I was able to make it work by using TextDocumentContentProvider to render my html/js. I didn't need to modify anything outside of the html/js files I was rendering.

Reply:
Ok, seems WebViews is the "new" way to go based on this https://code.visualstudio.com/updates/v1_21#_webview-api